cypresse2e/cypress/integration/e2eTests/requestHelpQuestion/requestHelpQuestion.spec.js

```diff
@@ -71,9 +71,10 @@ describe('FightPandemics Request Help Questionnaire', () => {
 
         it('User can type in location field and see a list of available locations', () => {
             var locationField = requestHelpQuestion.getRhLocationField();
-            locationField.should('be.visible').click();
-            locationField.type(LOCATION);
-            requestHelpQuestion.getRhLocationDropdown().should('be.visible');
+            locationField.should('be.visible').click({ force: true });
+            locationField.type(LOCATION)
+            .and('have.attr', 'aria-expanded', 'true')
+            .and('have.attr', 'aria-activedescendant', 'RES2_EAZ_list_0').click();
```

Comment:
I'm getting the resizeObserver error when running the test with this change. With that being said, it seems like if I tweak the ignore resizeObserver error code you already added in the index.js to the exact format of what's shown in the screenshot, I don't run into this error anymore for some reason. Maybe we can change the format in the index.js to reflect this.

![image](https://user-images.githubusercontent.com/56090896/95605508-0cd1a400-0a27-11eb-9793-8a320a0a07ee.png)

Reply:
@jessica-lau 

Originally I saw this error:
![Screen Shot 2020-10-09 at 14 30 16](https://user-images.githubusercontent.com/21987083/95607507-1c1f1480-0a5c-11eb-9cee-82ff98600b9d.png)
So I added forced:clicked but then the rest of the test failed. Maybe we can keep:
```
it('User can type in location field and see a list of available locations', () => {
            var locationField = requestHelpQuestion.getRhLocationField();
            locationField.should('be.visible').click();
            locationField.type(LOCATION);
            requestHelpQuestion.getRhLocationDropdown().should('be.visible');

```
But we will have to change this line of code:  locationField.should('be.visible').click();

Reply:
Yes, I agree with keeping the original test for requestHelp and just changing the line of code above.